tests/SprykerEcoTest/Zed/Unzer/_support/UnzerZedTester.php

```diff
@@ -188,6 +196,11 @@ class UnzerZedTester extends Actor
      */
     protected const CURRENCY_CODE = 'EUR';
 
+    /**
+     * @var \SprykerEco\Zed\Unzer\Business\UnzerFacadeInterface
+     */
+    protected $facade;
```

Comment:
several constants at this class are unused
cleanup them

Reply:
Fixed.